install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,34 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes have labels that often indicate their region. A convention is to set the
+`*region=infra*` label for the router and registry selector. For example:
```

Comment:
@ahardin-rh I think the default setting is a little confused here. Above, you're telling the reader to set the region=infra label for the router and registry, but below, you're telling them that nodes have the label by default. Should the below say "All new nodes will have the *region=infra* label by default"?